.github/workflows/nightly_dumps.yml

```diff
@@ -0,0 +1,47 @@
+name: CLT testing of supported versions of mysqldump
+run-name: CLT testing of supported versions of mysqldump ${{ github.sha }}
+
+on:
+  schedule:
+    - cron: '00 20 * * *'
+  push:
+    paths:
+      - 'test/clt-tests/**'
+    branches:
+      - manticore-6.2.12
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'test/clt-tests/mysqldump/versions/**'
+
+# cancels the previous workflow run when a new one appears in the same branch (e.g. master or a PR's branch)
+concurrency:
+  group: clt_nightly_${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
I'm not sure about this. I am afraid this block will prevent running `install nithgtly` tests. You should check this

Reply:
Change